ui_framework/doc_site/src/views/local_nav/local_nav_example.js

```diff
@@ -125,6 +128,29 @@ export default props => (
     </GuideSection>
```

Comment:
My guess is that it doesn't recognize JSX or, if it does, it's trying to highlight invalid syntax because of the closing tag at the top?